openprescribing/frontend/tests/functional/test_measures.py

```diff
@@ -319,7 +319,7 @@ def test_measures_for_one_practice(self):
         self._verify_link(
             panel_element,
             ".inner li:nth-child(3)",
-            "Compare all Sub-ICB Locations in England on this measure",
+            "Compare all SICBLs in England on this measure",
```

Comment:
How did these pass before?

Reply:
They didn't!

Reply:
Oh!  These tests are failing in CI.  Bamboozling.

Reply:
I wonder if you haven't rebuilt your JS locally?

Reply:
5am thought: out-of-date JS?

Reply:
jinx